gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -350,11 +352,6 @@ public WritableByteChannel create(final StorageResourceId resourceId, CreateObje
      *    later.
      *
      * To counter this we need to perform mutations with a condition attached, always.
-     *
```

Comment:
Add link to GCS doc that describes race-conditions that it fixes: https://cloud.google.com/storage/docs/generations-preconditions#preventing_the_race_condition